### ISSUE TITLE
Copy what's in Briefcase's docs

### ISSUE THREAD

**Issue by ahonnecke** [beeware/toga-chart#64]
<SNIP>
As for toga-chart's documentation, it probably wouldn't hurt to copy what's in Briefcase's docs....although, it looks like the docs here could use a little TLC; for instance, I see a reference to Python 3.5 and it isn't using the [furo](https://github.com/pradyunsg/furo) theme.

_Originally posted by @rmartin16 in https://github.com/beeware/toga-chart/issues/61#issuecomment-1791393232_

**Comment by freakboy3742** [comment]
To clarify the scope for this as a ticket - the aim if this ticket is "write initial docs for toga-chart". The broad structure of docs for a BeeWare project is always the same - "tutorial, howto, topics, reference, about" - and toga-chart should follow that pattern. This includes:

* Use of the beeware theme additions to Furo
* A Read The Docs configuration
* tox configurations that have docs-live, docs-lint, and docs targets for local testing

In terms of content - it's OK for there to be lots of gaps in the first pass. We don't need to have any actual howtos or topics in the first pass. The FAQ can be very bare bones with some basic "positioning" questions - "What is toga-chart" is possibly all we need. At least one link to the matplotlib docs (probably on the topic guide page?) would be called for.

However, we *do* need a *basic* tutorial. It doesn't have to be very detailed, or document how to use matplotlib in detail - "add a chart as the only widget in an app" is all you need. Doing a line-by-line walkthrough of the example app in the toga-chart repo is probably sufficient.

**Comment by feoh** [comment]
@freakboy3742 I took a swing at this with Codex in https://github.com/beeware/toga-chart/pull/325

I built the HTML docs locally. Not sure of there's a way to trigger readthedocs to make a throw-away PR version we can review more easily.

I worked through the tutorial is generated, examined the Reference section, and carefully read the rest.

To my uneducated eye it looks good. I have a few questions at this point:
- The tutorial starts with the phrase "Start with a regular Toga application module".


It feels to me like this assumes a lot of our new users which is a bad idea.

- How do users install this?
- Should the tutorial go further? if so, in what direction? Where should I add detail?

Thanks!

**Comment by freakboy3742** [comment]
> [@freakboy3742](https://github.com/freakboy3742) I took a swing at this with Codex in [#325](https://github.com/beeware/toga-chart/pull/325)
> 
> I built the HTML docs locally. Not sure of there's a way to trigger readthedocs to make a throw-away PR version we can review more easily.

Looks like I neglected to turn on PR builds for ReadTheDocs; I've just turned that on, so going forward, PRs should generated a PR preview.

The current docs are all on the older Sphinx-based framework; the first step needed here will be to convert the docs to the MkDocs-based framework used by Briefcase, Toga etc.
 
> * The tutorial starts with the phrase "Start with a regular Toga application module".
> 
> It feels to me like this assumes a lot of our new users which is a bad idea.

Completely agreed. The current tutorial is... existent, but not much else good can be said about it :-)
 
> * How do users install this?

For the purposes of a tutorial, we can assume they're using a "stable" release - so adding `toga-chart` to a dependency list should be sufficient.

> * Should the tutorial go further? if so, in what direction? Where should I add detail?

I don't think the tutorial needs to go into depth into configuring and using Matplotlib, but it should leave the user in no confusion over where the handover from the Toga-chart tutorial and matplotlib tutorial/docs occurs.

**Comment by kattni** [comment]
This was resolved in #332, to the best of my knowledge. @freakboy3742 Is there further scope to this that I'm not seeing, or can this be closed?

**Comment by freakboy3742** [comment]
Agreed - we now have a scaffold of basic documentation that we can build on.
